src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -150,6 +150,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('file', options_list=['--file', '-f'], help="The task template/definition file path relative to the source context.")
         c.argument('values', help="The task values file path relative to the source context.")
         c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format.", action='append', validator=validate_set)
+        c.argument('set_secret', help="Secret value in 'name[=value]' format.", action='append', validator=validate_set_secret)
```

Comment:
Looks fine, could you add a test for `acr run`?

Reply:
Anywho, I added a test and the recording has the parameters translated correctly with a successful run. I assume that's enough even though the test doesn't do much.

Reply:
Generally, as a convention, our commands will output JSON by default. Since the current scenario is a valid one for users that we dont want to break, I think the current behavior with outputting the logs is fine.
I agree that testing will not give much value to the command.